document_merge_service/settings.py

```diff
@@ -165,6 +165,8 @@ def parse_admins(admins):
 AWS_LOCATION = env.str("AWS_LOCATION", "")
 AWS_S3_FILE_OVERWRITE = env.bool("AWS_S3_FILE_OVERWRITE", False)
 AWS_S3_SIGNATURE_VERSION = env.str("AWS_S3_SIGNATURE_VERSION", "v2")
+AWS_S3_USE_SSL = env.bool("AWS_S3_USE_SSL", default=True)
+AWS_S3_VERIFY = env.bool("AWS_S3_VERIFY", default=None)
```

Comment:
These should also be mentioned in the https://github.com/adfinis/document-merge-service/blob/ssl-settings/CONFIGURATION.md#django-storages-s3-settings section of the configuration docu

Reply:
Added